### PR TITLE
Mobilizon federation

### PR DESCRIPTION
**Pull request by Nutomic**
Some minor changes to be able to parse events.

### CODE REVIEW

File: crates/apub/src/objects/post.rs [hunk 1]
```diff
@@ -179,11 +179,12 @@ impl ApubObject for ApubPost {
...
       let url = if let Some(attachment) = page.attachment.first() {
-        Some(match attachment {
+        match attachment {
           // url as sent by Lemmy (new)
-          Attachment::Link(link) => link.href.clone(),
-          Attachment::Image(image) => image.url.clone(),
-        })
+          Attachment::Link(link) => Some(link.href.clone()),
+          Attachment::Image(image) => Some(image.url.clone()),
+          Attachment::Other(_) => None,
```
Comment (Nutomic) [inline crates/apub/src/objects/post.rs:186]: This is wrong.

Comment (Nutomic) [reply crates/apub/src/objects/post.rs]: fixed

**Comment by dessalines** [comment]
> Checking lemmy_db_views_moderator v0.16.5 (/drone/src/crates/db_views_moderator)
error: failed to write query cache to `/drone/src/target/x86_64-unknown-linux-musl/debug/incremental/lemmy_db_views_moderator-2lv1bjrqqfo4y/s-gf9dpwe0hk-1dy5n71-working/query-cache.bin`: No space left on device (os error 28)

**Review by dessalines — APPROVED** [review]

**Comment by Nutomic** [comment]
Thats probably one of your builders, mine has plenty of free space (and a cronjob to clean up old docker images).

**Comment by dessalines** [comment]
My builder isn't running rn, but I'll also run docker system prune -a

**Review by dessalines — APPROVED** [review]

**Pull request merged**